frontend/src/views/WheresMyGene/components/GeneSearchBar/components/SaveExport/index.tsx

```diff
@@ -151,10 +165,17 @@ export default function SaveExport({
 
     setDownloadStatus({ isLoading: true, blur: true });
 
-    const heatmapNode = document.getElementById("view") as HTMLDivElement;
+    const heatmapNode = document.getElementById(
+      "heatmap-container-id"
+    ) as HTMLDivElement;
+    const viewNode = document.getElementById("view") as HTMLDivElement;
```

Comment:
we need to change the observer target node since we now render the message banner while downloading, in order for the banner to render in the exports. The banner is a child of the container with id="view", so adding and removing the banner will trigger the observer twice, thus causing the file to be downloaded twice.

We only want to listen to changes to the heatmap anyways, so we have the observer listen to the container with id `heatmap-container-id`